_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme

```diff
@@ -413,79 +355,56 @@ function rhdp2_preprocess_assembly(&$variables) {
         $audience_selection_values[] = $field_audience_selection_value['value'];
       }
 
+      // @TOD: This should be passed to the attributes array.
```

Comment:
Yeah, I agree that it makes more sense to just add the `data-audience` data attribute to the `#attributes` array. We could do that in this PR if you'd like